components/automate-cli/cmd/chef-automate/verify.go

```diff
@@ -606,6 +606,18 @@ func buildReports(batchCheckResults []models.BatchCheckResult) []reporting.Verif
 	for _, batchCheckResult := range batchCheckResults {
 
 		for _, test := range batchCheckResult.Tests {
+
+			var errorMsgs, resolutionMsgs []string
+			var successfulCount, failedCount int
+
+			if test.Skipped {
```

Comment:
I think line 613 will cause problem to print skipped checks on UI

Reply:
done